packages/relay/src/lib/poller.ts

```diff
@@ -76,6 +76,12 @@ export class Poller {
           this.logger.error(`${LOGGER_PREFIX} Polling for unsupported event: ${event}. Tag: ${poll.tag}`);
         }
 
+        if (event === 'newHeads' && process.env.WS_NEW_HEADS_ENABLED === 'true') {
```

Comment:
is there a test case when `WS_NEW_HEADS_ENABLED=false`?

Reply:
Added test